src/shared/api/index.ts

```diff
@@ -10,6 +11,8 @@ interface HTTPService {
   fetchPeerStatus(): Promise<Status>;
   fetchRoles(): Promise<Role[]>;
   fetchBlocks(params?: PaginationParams): Promise<Paginated<BlockShallow>>;
+  fetchBlocksDetails(id: number): Promise<Block>;
+  fetchBlockTransactions(transactions:TransactionDto[], params?: PaginationParams):Promise<Paginated<TransactionDto>>;
```

Comment:
`fetchBlockTransactions` - what API endpoint does it reflect?

This interface is a convenient wrap around backend's API. AFAIK, backend doesn't have a method to get transactions from a certain block.

Reply:
At first was using this coupled with the composable to return a ```paginated transactions wrap``` , it no longer serves any purpose.